### ISSUE TITLE
Weird Bug in Property Assignment

### ISSUE THREAD

**Issue by SergioSaavi** [rubenvitt/readwise-to-mem.ai-exporter-using-notion#5]
I am reaching out to report an unusual bug that I have encountered while working with your tool. 

I've been following the documentation precisely and have set up attributes like “memai-sync-status”, “memai-sync-id”, and “memai-last-sync” in my Notion database as instructed.

However, I’m running into a validation error that I haven’t been able to resolve. The error message is as follows:

code: 'validation_error',
message: 'memai-sync-status is expected to be status.'

I have ensured that the “memai-sync-status” attribute is set up as a Status Property 

I am curious if there might be an underlying issue or if there's something specific I am missing. Any insights or assistance you could provide would be greatly appreciated. I am looking forward to leveraging this tool to its full capacity and am eager to resolve this issue promptly.

Thank you in advance for your time and support.

**Comment by SergioSaavi** [comment]
I am pleased to inform you that I have managed to resolve the issue I previously encountered. However, I have noticed an inconsistency in the documentation that could potentially lead to confusion for other users.

In the current documentation, it is stated that memSyncStatus should be a Status property. However, my findings indicate that it should actually be a Select property, with "SYNCED" being one of the selectable options to indicate the completion of the synchronization process.

Here is the corrected code snippet for clarity:

javascript
Copy code
properties[propertyNames.memSyncStatus] = {
  select: {
    name: 'SYNCED',
  },
};
I believe an update to the documentation to reflect this correction would be highly beneficial for future users to avoid similar issues and ensure a smooth user experience.
